tests/unit/test_bronze_tier_compliance.py

```
@@ -218,6 +218,71 @@ async def test_appropriate_polling_interval(self, hass, api_ok):
             "DEFAULT_SCAN_INTERVAL" in source or "scan_interval" in source
         ), "Coordinator __init__ should reference scan_interval"
 
+    async def test_branding_assets_exist(self):
+        """Test: brands - Has branding assets available.
+
+        Bronze tier requires icon.png and icon@2x.png files.
+        As of HA 2026.3.0, custom integrations can include icons directly.
+
+        TODO: Download Imou logo and create icon files:
+        - icon.png (256×256) and icon@2x.png (512×512)
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)